src/Module/Help.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-tooling for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-tooling/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Tooling\Module;
+
+use Zend\Stdlib\ConsoleHelper;
+
+class Help
+{
+    const TEMPLATE = <<< 'EOT'
+<info>Usage:</info>
+
+  %s <command> [options] modulename
+
+<info>Commands:</info>
+
+  <info>help</info>          Display this help/usage message
+  <info>create</info>        Create skeleton of the expressive module and register it
+  <info>register</info>      Register module in configuration and composer autoloading
```

Comment:
"Register module in application configuration, and enable autoloading of module via composer."